content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md

```diff
@@ -36,7 +36,7 @@ LoadBalancer, or with dynamic PersistentVolumes.
 For both methods you need this infrastructure:
 
 - Three machines that meet [kubeadm's minimum requirements](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#before-you-begin) for
-  the masters
+  the control plane nodes
```

Comment:
@devurandom `control-plane` is more widely used, so better have with it with the dash in this document too:
https://github.com/kubernetes/website/search?q=%22control-plane%22&unscoped_q=%22control-plane%22

Reply:
I get 476 results for "control plane", which includes "control-plane": https://github.com/kubernetes/website/search?q=%22control+plane%22&unscoped_q=%22control+plane%22
And 205 results for "control-plane": https://github.com/kubernetes/website/search?q=%22control-plane%22&unscoped_q=%22control-plane%22

So it is actually quite even. I will undo the change from "control-plane" to "control plane" in this document.